runtime/doc/lua.txt

```diff
@@ -2697,14 +2697,14 @@ There are four separate window types used by this interface:
 • "cmd": The cmdline window; also used for 'showcmd', 'showmode', 'ruler', and
   messages if 'cmdheight' > 0.
 • "box": The message box window; used for messages when 'cmdheight' == 0.
-• "more": The more-prompt window; used for |:messages| and certain messages
-  that should be shown in full.
-• "prompt": The cmdline prompt window; used for prompt messages that expect
-  user input.
-
-These four windows are assigned the "cmdline", "msgbox", "msgmore" and
-"msgprompt" 'filetype' respectively. Use a |FileType| autocommand to configure
-any local options for these windows and their respective buffers.
+• "dialog": The dialog window; used for prompt messages that expect user
+  input.
+• "pager": The pager window; used for |:messages| and certain messages that
+  should be shown in full.
+
+These four windows are assigned the "Cmdline", "Messages", "Dialog" and
+"Pager" 'filetype' respectively. Use a |FileType| autocommand to configure any
```

Comment:
filetype names are usually lowercase. though the buffer names would pascalcase (`[Cmdline]`, etc)

Reply:
Sure, changing that in #34378 then. Will rebase when that is merged.

Reply:
just for reference: it looks like there is a [`messages` filetype](https://github.com/neovim/neovim/blob/d86d4bacc14bc6ab51b3173f3644d703f53ffbde/runtime/lua/vim/filetype.lua#L2080-L2081) already :(  so we may need to think of a different filetype name for that one. can name it `messages2` and revisit later.